### PR TITLE
 networking/samba

### PR DESCRIPTION
**Pull request by jmazanek**
- message printed regarding the deprecation of SMB1 protocol

### CODE REVIEW

File: RHEL6_7/networking/samba/check [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+. /usr/share/preupgrade/common.sh
+
+#END GENERATED SECTION
+
+
+SOLUTION_FILE="./solution.txt"
```
Comment (pirat89) [inline RHEL6_7/networking/samba/check]: Why do you create this variable for one use in echo? It is unneeded.

File: RHEL6_7/networking/samba/check [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+echo "SMB1 protocol is deprecated as it is replaced by SMB2 protocol. It is now considered insecure, due to it's vulnerability to man-in-the-middle attacks. Support for the SMB2 protocol will not be backported to Red Hat Enterprise Linux 6 version of samba\* packages. Therefore, for security and compatibility reasons, we strongly recommend to the user to upgrade whole Red Hat Enterprise Linux based server - clients network at once." > $SOLUTION_FILE
```
Comment (pirat89) [inline RHEL6_7/networking/samba/check]: When the message is persistent and there is not any other check than the samba-common package available on system, the message shoud be part of the solution.txt file.

File: RHEL6_7/networking/samba/check [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+log_high_risk "SMB1 protocol is deprecated."
+exit $RESULT_FAIL
```
Comment (pirat89) [inline RHEL6_7/networking/samba/check]: High risk is not needed here as the action is required on the target machine. Use rather the `log_medium_risk` instead.

**Review by pirat89 — CHANGES_REQUESTED** [review]

**Comment by jmazanek** [comment]
Module has been edited in accordance with requested changes.  Message text has been already reviewed by jhornice. Please merge

**Review by pirat89 — DISMISSED** [review]
It seems OK. I'll keep it opened as changes in text are expected yet and it can be merge everytime before the next release.

**Comment by jmazanek** [comment]
The link to the article discussing in detail implications of the change for the interworking with Windows based networks was added to the module

**Review by pirat89 — APPROVED** [review]

**Pull request merged**